cmd/bbolt/main.go

```diff
@@ -1312,19 +1312,22 @@ func (cmd *BenchCommand) Run(args ...string) error {
 	defer db.Close()
 
 	// Write to the database.
-	var results BenchResults
-	if err := cmd.runWrites(db, options, &results); err != nil {
+	writeResults := BenchResults{int64(0), 0}
+	fmt.Fprintf(cmd.Stdout, "starting write benchmark.\n")
```

Comment:
Why not to stick to a single output stream (i.e. Stderr) ?

Reply:
update, use Stderr as single output stream